Repository.playground/Contents.swift

```diff
@@ -32,8 +72,23 @@ class User: Codable {
 let userRepo = UserRepository(withPath: "http://216.186.69.45/services/device/users/")
 
 //Fetch a single User
-userRepo.fetch(withId: 43, withCompletion: {(user) in
+userRepo.fetch(withId: 58, withCompletion: {(user) in
         print(user!.FirstName ?? "no user")
 })
 
+// Create (post) new user
+var mUser = User()
+mUser.FirstName = "jim"
+mUser.LastName = "beam"
```

Comment:
James - I think the issue with your code not running is that some of your user properties are nil (PhoneNumber and SID)

Reply:
Otherwise, the methods look good